step/src/main/java/com/google/sps/CalendarManager.java

```diff
@@ -0,0 +1,150 @@
+      .setStartDateTime(year, month, day, hour, minute)
+      .build();
+
+    pushMatchEvent(hostUser, guestUser, matchEvent);
+  }
+
+  public static List<String> getScopes() {
+    List<String> scopes = new ArrayList<>();
+    scopes.add(CalendarScopes.CALENDAR_EVENTS); // "View and edit events on all your calendars"
+    return scopes;
+  }
+
+  // hostUser is the event owner and guestUser will receive an email invite
+  private static void pushMatchEvent(User hostUser, User guestUser, Event event) {
+    Calendar calendar = getCalendar(hostUser);
+    try {
+      String calendarId = "primary";
+      event = calendar.events().insert(calendarId, event).execute();
+      System.out.printf("Event created: %s\n", event.getHtmlLink());
+    } catch (Exception e) {
+      System.out.println("Unable to push match event.");
+      e.printStackTrace();
+    }
+  }
+
+  private static Calendar getCalendar(User user) {
+    return new Calendar.Builder(
+        new NetHttpTransport(),
+        new JacksonFactory(),
+        user.getCredential()
+      ).setApplicationName("Friend Matching Plus").build(); // TODO: not sure if app name matters
+  }
+
+}
+
+class MatchEventBuilder {
+  private Event matchEvent;
+
+  public MatchEventBuilder() {
+    matchEvent = new Event();
+  }
+
+  public MatchEventBuilder setAttendees(User hostUser, User guestUser) {
+    matchEvent.setSummary("FMP: " + hostUser.getName() + " / " + guestUser.getName());
+
+    matchEvent.setDescription(hostUser.getName() + " and " + guestUser.getName()
+                              + " have matched on Friend Matching Plus!");
+
+    matchEvent.setAttendees(Arrays.asList(new EventAttendee[] {
+      new EventAttendee().setEmail(guestUser.getEmail())
+    }));
+
+    return this;
+  }
+
+  // ZonedDateTime guide: https://www.baeldung.com/java-8-date-time-intro
+  public MatchEventBuilder setStartDateTime(int year, int month, int day, int hour, int minute) {
+    String dateTimeString = dateTimeString(year, month, day, hour, minute);
+
+    EventDateTime start = new EventDateTime()
+      .setDateTime(new DateTime(dateTimeString));
+      // .setTimeZone("America/Los_Angeles"); // TODO how to configure time zones?
+    matchEvent.setStart(start);
+
+    // Assume a 1 hour event.
+    DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ISO_OFFSET_DATE_TIME;
+    ZonedDateTime zonedDateTime = ZonedDateTime.parse(dateTimeString, dateTimeFormatter);
+    zonedDateTime = zonedDateTime.plusHours(1);
+    dateTimeString = zonedDateTime.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+
+    EventDateTime end = new EventDateTime()
+      .setDateTime(new DateTime(dateTimeString));
+    matchEvent.setEnd(end);
+
+    return this;
+  }
+
+  public Event build() {
+    return matchEvent;
+  }
+
+  // createDateTimeString(2020, 07, 21, 9, 0, 0) -> "2020-07-21T09:00:00-05:00"
+  private static String dateTimeString(int year, int month, int day, int hour, int minute) {
```

Comment:
Timezones should be more in "American/Los_Angeles" type of format. This is not a good representation and prone to errors. Particularly with daylights savings and such. Feel free to talk to me more about time representation in Java. I have done some complex time stuff in Java in the past.